src/ahrens_lab_to_nwb/yu_mu_cell_2019/yu_mu_cell_2019_imaging_extractor.py

```diff
@@ -29,22 +32,36 @@ def __init__(self, file_path: PathType, sampling_frequency: float):
     def get_video(self, start_frame: Optional[int] = None, end_frame: Optional[int] = None) -> np.ndarray:  # noqa: D102
         with h5py.File(name=self.file_path) as file:
             video = DatasetView(file["default"]).lazy_transpose(self._frame_axis_order)
-            return video.lazy_slice[:, :, :].dsetread()[np.newaxis, :]
 
-    def get_image_size(self) -> Tuple[int, int, int]:  # noqa: D102
-        return (self._num_rows, self._num_cols, self._num_stacks)
+            if self.region is None:
+                region_slice = slice(None)
+            elif self.region == "top":
+                region_slice = slice(int(self._num_rows / 2), None)
+            elif self.region == "bottom":
+                region_slice = slice(None, slice(int(self._num_rows / 2), None))
```

Comment:
For the dual-color data, the images are 2048 x 2048 but the top 1024 x 2048 and bottom 1024 x 2048 technically correspond to a different optical channel, so splitting them into different TwoPhotonSeries with this new argument.

The single-color data are not to be partitioned like this, though.